core/lib/Thelia/Model/FeatureProductQuery.php

```diff
@@ -16,5 +17,14 @@
  */
 class FeatureProductQuery extends BaseFeatureProductQuery
 {
+    /**
+     * @deprecated
+     */
+    public function filterByFreeTextValue($freeTextValue = null, $comparison = null)
+    {
+        $bt = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 1);
+        Tlog::getInstance()->warning(sprintf('Using `free_text_value` is deprecated. Use `is_free_text` instead. Used in %s:%d', $bt[0]['file'], $bt[0]['line']));
+        return parent::filterByFreeTextValue($freeTextValue, $comparison);
+    }
 }
```

Comment:
Please deprecate also all *FreeTextValue() methods, such as orderByFreeTextValue(), findOneByFreeTextVaue(), and so on.